reconcile/utils/gql.py

```diff
@@ -29,6 +30,19 @@
 """
 
 
+def capture_and_forget(error):
```

Comment:
is this what you had in mind @rporres?

Reply:
I guess that makes me feel more relaxed about this  :-D